WordPress/wordpress-playground
# WordPress: Keep dashicons, icons, and Akismet CSS in the bundle

playground.wordpress.net ships an optimized WordPress build where most
static assets are removed to keep the initial download snappy. However,
some backend code actually depends on those assets being present as seen
in https://github.com/WordPress/wordpress-playground/issues/729 and https://github.com/WordPress/wordpress-playground/issues/770. This PR allowlists additional files:

* wp-includes/images/media
* wp-includes/fonts/dashicons.svg
* wp-content/plugins/akismet/_inc

Closes #729
Closes #770

## Testing instructions

To test, run Playground locally, go to this URL, and confirm it displays
the list of files as shown below.

http://localhost:5400/website-server/#%7B%22landingPage%22:%22/test.php%22,%22steps%22:[%7B%22step%22:%22writeFile%22,%22path%22:%22/wordpress/test.php%22,%22data%22:%22%3Cpre%3E%3C?php%20var_dump(glob(%27/wordpress/wp-includes/fonts/*%27));%20var_dump(glob(%27/wordpress/wp-includes/images/media/*%27));%20var_dump(glob(%27/wordpress/wp-content/plugins/akismet/_inc/*%27));%22}]}


```
array(3) {
  [0]=>
  string(60) "/wordpress/wp-includes/fonts/class-wp-font-face-resolver.php"
  [1]=>
  string(51) "/wordpress/wp-includes/fonts/class-wp-font-face.php"
  [2]=>
  string(42) "/wordpress/wp-includes/fonts/dashicons.svg"
}
array(9) {
  [0]=>
  string(47) "/wordpress/wp-includes/images/media/archive.png"
  [1]=>
  string(45) "/wordpress/wp-includes/images/media/audio.png"
  [2]=>
  string(44) "/wordpress/wp-includes/images/media/code.png"
  [3]=>
  string(47) "/wordpress/wp-includes/images/media/default.png"
  [4]=>
  string(48) "/wordpress/wp-includes/images/media/document.png"
  [5]=>
  string(51) "/wordpress/wp-includes/images/media/interactive.png"
  [6]=>
  string(51) "/wordpress/wp-includes/images/media/spreadsheet.png"
  [7]=>
  string(44) "/wordpress/wp-includes/images/media/text.png"
  [8]=>
  string(45) "/wordpress/wp-includes/images/media/video.png"
}
array(3) {
  [0]=>
  string(59) "/wordpress/wp-content/plugins/akismet/_inc/akismet-admin.js"
  [1]=>
  string(62) "/wordpress/wp-content/plugins/akismet/_inc/akismet-frontend.js"
  [2]=>
  string(53) "/wordpress/wp-content/plugins/akismet/_inc/akismet.js"
}
```
